.github/workflows/release.yml

```diff
@@ -734,11 +777,11 @@ jobs:
             owner: context.repo.owner,
             repo: context.repo.repo,
             version,
-          });
+          }).catch(console.error);
```

Comment:
we shouldn't ever fail the job because of an issue with milestone closing/creation